README.md

```diff
@@ -119,20 +120,21 @@
 ## Compiling
 
 - To compile with default settings, run `./build.sh`
-- For versions of Ubuntu with custom versions of gcc, but did not set the default compiler with update-alternatives:
+- For those who want to modify the default configuration and compile themselves, we use the following:
 
     ``` shell
-    CC=gcc-11 CXX=g++-11 cmake -S . -Bbuild -GNinja
+    cmake -S . -Bbuild -GNinja
     cmake --build build
     ```
 
```

Comment:
Check if there is an extra blank line after this shell snippet.

Reply:
There was, in fact, an extra space. It has been removed in the latest commit on this branch.

Reply:
Accidentally marked this comment on the wrong line. It actually was lines 136-137 in this commit.